packages/tic-tac-toe/contracts/TicTacToeHelpers.sol

```diff
@@ -0,0 +1,91 @@
+pragma solidity ^0.5.11;
```

Comment:
This file seems to be unused, and replicated inside `TicTacToe.sol`. Shall we remove it?